### DISCUSSION TITLE
后来支持“全部导入，或多选导入”了吗？

### DISCUSSION THREAD

**Discussion by allen7u** [MuiseDestiny/zotero-reference#326] [General]
如题，如果支持的话，怎么做？目前在用 Zotero 6 的 0.58 版，没看到相关选项

**Comment by MuiseDestiny** [comment]
Zotero 7 已经支持。无更新Zotero 6版本插件计划。
![image](https://github.com/user-attachments/assets/53ef5947-8190-406e-be45-85a83773872f)

多选全部选中 导入
